src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -70,19 +74,21 @@ const WorkspaceTabs = ({
 
 const WorkspaceContainer = ({
   namespace, name, breadcrumbs, topBarContent, title, activeTab, showTabBar = true, refresh, refreshRuntimes, workspace,
-  runtimes, persistentDisks, appDataDisks, apps, refreshApps, location, locationType, children
+  refreshWorkspace, runtimes, persistentDisks, appDataDisks, apps, refreshApps, location, locationType, children
 }) => {
   const [deletingWorkspace, setDeletingWorkspace] = useState(false)
   const [cloningWorkspace, setCloningWorkspace] = useState(false)
   const [sharingWorkspace, setSharingWorkspace] = useState(false)
+  const [showLockWorkspaceModal, setShowLockWorkspaceModal] = useState(false)
 
   return h(FooterWrapper, [
     h(TopBar, { title: 'Workspaces', href: Nav.getLink('workspaces') }, [
       div({ style: Style.breadcrumb.breadcrumb }, [
         div({ style: Style.noWrapEllipsis }, breadcrumbs),
         h2({ style: Style.breadcrumb.textUnderBreadcrumb }, [
           title || `${namespace}/${name}`,
-          workspace && !Utils.canWrite(workspace.accessLevel) && span({ style: { paddingLeft: '0.5rem', color: colors.dark(0.85) } }, '(read only)')
+          workspace && !Utils.canWrite(workspace.accessLevel) && span({ style: { paddingLeft: '0.5rem', color: colors.dark(0.85) } }, '(read only)'),
+          workspace && workspace.workspace.isLocked && span({ style: { paddingLeft: '0.5rem', color: colors.dark(0.85) } }, '(locked)')
```

Comment:
@MatthewBemis I imagine you were following the pattern of the "(read only)" text, but this color does not meet color contrast requirements and is very difficult to read. I'm going to make it the same color as the workspace name.

![image](https://user-images.githubusercontent.com/484484/158703765-d3e25418-3d89-4f77-84c4-a87560acec5c.png)

Reply:
Yeah exactly- it was based off of the "(read only)" text. Thanks for calling this out- it is indeed a bit hard to read, particularly with certain monitor settings.

And yes it's possible for them both to happen at the same time.

Reply:
@MatthewBemis I fixed the accessibility issue in https://github.com/DataBiosphere/terra-ui/pull/2882. I did not combine the text together though-- I certainly think it's worth doing, but it's not as serious a problem as not being able to read the text.

Reply:
For the future: `workspace?.workspace.isLocked`

Reply:
We now have a much nicer design for this, recorded in https://broadworkbench.atlassian.net/browse/WOR-217.